pkg/secrets/fetch_secret.go

```diff
@@ -58,15 +65,29 @@ func execCommand(inputPayload string) ([]byte, error) {
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 
+	start := time.Now()
 	err := cmd.Run()
+	elapsed := time.Since(start)
+	log.Debugf("secret_backend_command '%s' completed in %d ms", secretBackendCommand, elapsed.Milliseconds())
```

Comment:
Could format time.Duration instead of Milliseconds as int here?

Reply:
Good point ! 👍🏻 
Change done: https://github.com/DataDog/datadog-agent/pull/6381/commits/982a4a1a5afc13f36fb87dbb249fab4ee158310c